src/test/regress/sql/multi_deparse_function.sql

```diff
@@ -67,7 +67,9 @@ CREATE FUNCTION add(integer, integer) RETURNS integer
     LANGUAGE SQL
     IMMUTABLE
     RETURNS NULL ON NULL INPUT;
+SET citus.enable_object_propagation TO OFF;
```

Comment:
Can't we just set at the start of the file until `-- clear objects`? Would be good to have a comment at least one of the lines on why we set it to false

Reply:
I've added a comment as the same trick has applied multiple times. Having them multiple times looks clear to me, otherwise I got confused its' effect on other commands in the test.